src/context/config.test.tsx

```diff
@@ -20,7 +20,9 @@ const TestComponent: FunctionComponent = () => {
   const { config, setConfigValue } = useConfigContext();
   return (
     <View>
-      <Text testID="printed-network">{config.network}</Text>
+      <Text testID="printed-network">
+        {config ? config.network : undefined}
```

Comment:
When will config be falsy?

Reply:
Oh this was from a previous change when config was first unset then upon loaded with be displayed. Should I revert this change?

Reply:
Yep do revert

Reply:
👍